packages/chrome/src/elements/sheet/Sheet.tsx

```diff
@@ -67,16 +67,27 @@ export const Sheet = React.forwardRef<HTMLElement, ISheetProps>(
     const sheetRef = useRef<HTMLElement>(null);
 
     const seed = useUIDSeed();
+    const [isCloseButtonPresent, setCloseButtonPresent] = useState<boolean>(false);
```

Comment:
I think because the `Sheet.Close` button is optional was why the decision was made for padding to be conditional, otherwise it makes sense to do so.Another challenge that makes this nonstandard is that the `Sheet.Close` doesn't nest within the `Sheet.Header` container, which makes it tricky.